ui/apps/platform/src/Containers/DeferralConfiguration/DeferralConfigurationPage.tsx

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
-import { PageSection, Title } from '@patternfly/react-core';
+import { PageSection, Tab, Tabs, Title } from '@patternfly/react-core';
 
 import PageTitle from 'Components/PageTitle';
+import useURLStringUnion from 'hooks/useURLStringUnion';
+import VulnerabilitiesConfigurationContainer from './VulnerabilitiesConfigurationContainer';
+
+const deferralConfigurationCategories = ['Vulnerabilities'] as const;
 
 function DeferralConfigurationPage() {
+    const [category, setCategory] = useURLStringUnion('category', deferralConfigurationCategories);
```

Comment:
Oh, I didn't know about this hook. I was initially a little confused about what it did from just the name

Reply:
Hmm we could rename it, what would be a better name?
`useURLStringChoice`
`useURLEnum`
`useURLOption`